core/src/main/java/com/graphhopper/routing/util/parsers/helpers/OSMValueExtractor.java

```diff
@@ -79,7 +89,7 @@ public static double stringToMeter(String value) {
         value = INCH_PATTERN.matcher(value).replaceAll("in");
         value = FEET_PATTERN.matcher(value).replaceAll("ft");
         if (isInvalid(value))
-            throw new NumberFormatException("Cannot parse value for 'meter': " + value);
+            return Double.NaN;
```

Comment:
As we do not log these invalid cases I think we should get rid also of the code above:

```
if (enableLog) {
  LOG.warn("Unable to extract tons from malformed road attribute '{}' for way (OSM_ID = {}).", rawValue, way.getId());
}
```

WDYT?

Reply:
I'm not a fan of useless log output and this one qualifies IMHO. Nobody is going to wade through thousand lines of messages which merely tell you that the values of certain tags are malformed.

Reply:
> Nobody is going to wade through thousand lines of messages which merely tell you that the values of certain tags are malformed.

Just a note: I do this. I filter the logs for things I am interested in and then I fix it in OSM where there is a clear tagging error. I try to take an hour from time to time to check the logs and fix the worst things. I probably fixed several hundred tags and I think others might do the same.

Reply:
But maybe this should be part of a "clean-up OSM" website, where multiple contributors could improve mistakes without running GraphHopper? There are also many other problems that I would find ugly if we print them all into the logs like e.g. #1807 or subnetworks in general.

Reply:
> But maybe this should be part of a "clean-up OSM" website

Very good idea. Projects like [StreetComplete](https://github.com/westnordost/StreetComplete#screenshots) are better suited for this because they can be used by contributors without technical background.

Reply:
> But maybe this should be part of a "clean-up OSM" website, where multiple contributors could improve mistakes without running GraphHopper?

Sure, that would be even better :). We should think about building one or maybe contributing to an existing website to also show routing related information. 

> I would find ugly if we print them all into the logs like e.g. #1807 or subnetworks in general.

Yes, maybe we only do this for small regions or maybe configurable or something like this :)

Reply:
I already thought about using the data provided by [taginfo](https://taginfo.openstreetmap.org/keys/maxweight#values) to check the coverage of our extraction logic.